### PR TITLE
Improve installation instructions

### LINKED ISSUE #579: Default installation does not have optional dependencies
### Description

I'm following the documentation to install Calliope using 
```
mamba create -n calliope -c conda-forge/label/calliope_dev -c conda-forge calliope
```

However, the installation does not come with some dependencies such as plotly or GLPK. It seems to me that they belongs to optional dependencies of the dev version, but using the above command does not install those. 

### Related links

https://calliope.readthedocs.io/en/v0.7.0.dev3/installation/

### Version

Trying to install v0.7.0 dev3

### Proposed change

Change the command so Calliope actually comes with the required dependencies.

### PR DESCRIPTION
**Pull request by sjpfenninger**
Fixes #579

Summary of changes in this pull request:

* Improve installation instructions
* Add `glpk` to recommended installation one-liner so at least one solver is always installed

Reviewer checklist:

- [ ] Test(s) added to cover contribution
- [ ] Documentation updated
- [ ] Changelog updated
- [ ] Coverage maintained or improved

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
 
 ## Recommended installation method
 
 The easiest way to get a working Calliope installation is to use the free `mamba` package manager, which can install all of the four things described above in a single step.
 
-To get `mamba`, [download and install the "Mambaforge" distribution for your operating system](https://mamba.readthedocs.io/en/latest/index.html) (using the version for Python 3).
+To get `mamba`, the most straightforward approach is to [download and install the "Miniforge" distribution for your operating system](https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html).
 
-With mamba installed, you can create a new environment called `calliope` with all the necessary modules, including the free and open source GLPK solver, by running the following command in a terminal or command-line window
+With mamba installed, you can create a new environment called `calliope` with Calliope as well as the free and open source GLPK solver. Run the following command in a terminal or command-line window:
```
Comment (brynpickering) [inline docs/installation.md:20]:
```suggestion
With mamba installed, you can create a new environment called `calliope` with Calliope as well as the free and open source GLPK solver. 
Run the following command in a terminal or command-line window:
```

**Review by brynpickering — APPROVED** [review]

**Comment by sjpfenninger** [comment]
@brynpickering While we're improving this, I am actually wondering if we should recommend `conda` by the default again given that it now uses libmamba as its backend anyway.

**Comment by brynpickering** [comment]
Have you run any comparisons on speed?

**Comment by sjpfenninger** [comment]
I haven't, and maybe we just leave this aside for now.

**Comment by sjpfenninger** [comment]
I've added a bit of text on using Anaconda as an alternative.

**Review by brynpickering — APPROVED** [review]

**Pull request merged**